src/style.css

```diff
@@ -1 +1,65 @@
 /* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header__wrap {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+}
+
+.li__link {
```

Comment:
your code won't pass the tests cause you don't have a height for link, or paddings